src/plugins/intel_cpu/src/nodes/tensoriterator.cpp

```diff
@@ -151,6 +152,7 @@ class BackEdgePortHelper : public PortMapHelper {
 
     void execute(dnnl::stream strm, int iter = -1) override {
         if (iter != 0) {
+            DEBUG_LOG(mem_holder_src.get_data_handle(), " -> ", mem_holder_dst.get_data_handle());
```

Comment:
I think we should avoid having DEBUG_LOG inside the execute calls, even for TensorIterator

Reply:
Could remove it.